### DISCUSSION TITLE
Issue with interactive applications over SSH (WARNING: terminal is not fully fonctional)

### DISCUSSION THREAD

**Discussion by freddydumont** [ghostty-org/ghostty#4077] [Feature Requests, Ideas] [answered]
When launching scrollable programs such as `man` or `git log` in a SSH session, I get the following error:

```sh
WARNING: terminal is not fully functional
Press RETURN to continue
```

Nothing happens when pressing the cursor up and down keys. It's not possible to scroll down.

This doesn't happen with wezterm or iTerm2, and it only happens over SSH.

**Comment by JakeWharton** [comment, marked as answer]
Did you do this? https://ghostty.org/docs/help/terminfo#ssh

**Reply by freddydumont** [reply]
`SetEnv TERM=xterm-256color` works. Thanks for the hint!

**Comment by T3sT3ro** [comment]
Ubuntu 26 — This happens also on localhost with `journalctl -b -1` on a freshly opened ghosty
